qa/tasks/mgr/dashboard_v2/test_rbd.py

```diff
@@ -40,3 +46,36 @@ def test_list(self):
         self.assertEqual(img2['obj_size'], 4194304)
         self.assertEqual(img2['features_name'],
                          'deep-flatten, exclusive-lock, fast-diff, layering, object-map')
+
+    @authenticate
+    def test_create(self):
+        data = {'pool_name': 'rbd',
+                'name': 'test_rbd',
+                'size': 10240}
+        self._post('/api/rbd', data)
+        self.assertStatus(201)
+
+        # TODO: change to GET the specific RBD instead of the list as soon as it is available?
+        ret = self._get('/api/rbd/rbd')
+        rbd_names = [rbd['name'] for rbd in ret['value'] if 'name' in rbd]
+        self.assertStatus(200)
+        self.assertIn('test_rbd', rbd_names)
+
+        self._rbd_cmd(['rm', 'rbd/test_rbd'])
```

Comment:
Why are you removing the image inside the test? the images will be removed when you delete the pools in `tearDownClass` method.

Reply:
Just to clean it up where it was created. I will remove this line.